lib/elixir/lib/module/locals_tracker.ex

```diff
@@ -156,7 +156,7 @@ defmodule Module.LocalsTracker do
   end
 
   # Collect all unused definitions based on the private
-  # given also accounting the expected amount of default
```

Comment:
I happened to be looking through these and noticed this sentence has other issues as well. If you're changing the line (and the one afterwards): "given, also accounting for the number of default clauses a private function has".

Reply:
I am disinclined from messing with that sentence because it probably has additional problems that I can't fix without understanding exactly what it's talking about; e.g. "a private function has" - in general, not the specific private function? "based on the private"? not "for the private"?

Reply:
In this case `private` seems to be the second argument, though I admit to not understanding its purpose.